src/localization/index.js

```diff
@@ -34,29 +34,32 @@ export const dispensingStrings = new LocalizedStrings(dispensingStringsJSON);
 
 export const LANGUAGE_CODES = {
   ENGLISH: 'gb',
-  KIRIBATI: 'gil',
   FRENCH: 'fr',
+  KIRIBATI: 'gil',
+  LAOS: 'la',
+  MYANMAR: 'my',
   PORTUGUESE: 'pt',
   SPANISH: 'es',
   TETUM: 'tl',
-  LAOS: 'la',
 };
 
 export const LANGUAGE_NAMES = {
   [LANGUAGE_CODES.ENGLISH]: 'English',
-  [LANGUAGE_CODES.KIRIBATI]: 'te taetae ni Kiribati',
   [LANGUAGE_CODES.FRENCH]: 'French',
+  [LANGUAGE_CODES.KIRIBATI]: 'te taetae ni Kiribati',
+  [LANGUAGE_CODES.LAOS]: 'Laos',
+  [LANGUAGE_CODES.MYANMAR]: 'Myanmar',
```

Comment:
yes please - would like to update the list. It has also bothered me that one of them is in the language `te taetae ni Kiribati` but the others are in English. It is fairly common for the language selection to be in the language to be selected - so that other language speakers don't need to be able to recognise their language in English in order to select it (though, there is the flag icon too).

Also, `te taetae..` should be Capitalised, like the others?